### PR TITLE
Add a Color Resource

### LINKED ISSUE #1048: [Add] Color Resource
### Site URL

https://colorpalettes.net/

### Description of the above provided site

Color Palettes provide a free collection of color palette ideas for all occasions.

### Category

Frontend

### Sub Category

colors

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Sriparno08**
## Changes proposed

I have added a color resource to the website.

## Screenshots

Screenshot 2023-06-17 204553

## Note to reviewers

This PR is part of GSSoC'23.

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
LGTM! :)

File: database/frontend/colors.json [hunk 1]
```diff
@@ -131,5 +131,12 @@
     "url": "https://vanschneider.com/colors",
     "category": "frontend",
     "subcategory": "colors"
+  },
+  {
+    "name": "ColorPalettes",
+    "description": "Color Palettes provide a free collection of color palette ideas for all use cases.",
```
Comment (rupali-codes) [inline database/frontend/colors.json:137]:
```suggestion
    "description": "Color Palettes provides a free collection of color palette ideas for professional and commercial use.",
```

**Comment by Sriparno08** [comment]
Fixed, @rupali-codes!

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Review by rupali-codes — APPROVED** [review]
lgtm, thanks

**Pull request merged**